doc/api/console.md

```diff
@@ -417,10 +417,101 @@ added: v0.1.100
 
 The `console.warn()` function is an alias for [`console.error()`][].
 
+## Inspector only methods
+The following methods are exposed by the V8 engine in the general API but are non-op unless used
```

Comment:
I'd spell out what you mean by `non-op` rather than trust that readers will know what that means. So instead of `are non-op`, perhaps `do not display anything` or something like that.